subt/zmq-subt-x2.json

```diff
@@ -219,10 +208,7 @@
 
               ["fromrospy.octomap", "octomap.octomap"],
               ["fromrospy.pose3d", "octomap.pose3d"],
-              ["octomap.waypoints", "app.waypoints"],
-
-              ["fromrospy.rgbd_front", "logimage.rgbd"],
```

Comment:
I am definitely "out of order" ... i.e. ignore this status :( ... removing wrong front/rear